packages/gatsby-theme-aio/gatsby-config.js

```diff
@@ -14,6 +14,47 @@ require('dotenv').config({
   path: `.env`
 });
 
+if (!process.env.ALGOLIA_APP_ID) {
+  console.error('Missed required parameter: ALGOLIA_APP_ID');
+}
+if (!process.env.ALGOLIA_WRITE_API_KEY) {
+  console.error('Missed required parameter: ALGOLIA_WRITE_API_KEY');
+}
+if (!process.env.REPO_NAME) {
+  console.error('Missed required parameter: REPO_NAME');
+}
+
+const AlgoliaQueryBuilder = require('./algolia/query-builder');
+const algoliaQueryBuilder = new AlgoliaQueryBuilder();
+const queries = algoliaQueryBuilder.build();
+
+const algoliaIndexName =
+  process.env.ALGOLIA_INDEX_NAME_SUFFIX === undefined
+    ? process.env.REPO_NAME
+    : process.env.REPO_NAME + process.env.ALGOLIA_INDEX_NAME_SUFFIX;
+
+// By default push data to real index
+let algoliaSkipIndexing = false;
```

Comment:
Oh sorry I think there's some confusion here. I meant the other way around. By default, dont push any data to real index.

Reply:
Exactly. There should be no Algolia indexing by default. This must be an explicit operation that WE control centrally somehow. Site admins should request indexing only when they need it. Because in addition to lots of errors across sites, as Stephan noted, there would be major cost consequences. As I’ve noted elsewhere (in the other PR review), if indexing is on by default, it would be very expensive for 20+ and the eventual 80+ sites that are constantly building within the CI/CD pipeline. The Algolia plan we are currently paying for (1,000,000 record changes and 5,000,000 operations per month) would be exhausted within a few short days, and the overage charges for the rest of the month would be extremely expensive. 😱

Reply:
Done https://github.com/adobe/gatsby-theme-aio/pull/368/files#diff-42e1aea6395cd5e4d7fc1c084e6202fb605848806a5afe7c62128c46b7dfab36R27